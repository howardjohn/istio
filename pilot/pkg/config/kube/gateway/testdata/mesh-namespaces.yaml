apiVersion: gateway.networking.k8s.io/v1alpha2
kind: HTTPRoute
metadata:
  name: same
  namespace: default
spec:
  parentRefs:
  - kind: Mesh
    name: istio
  hostnames: ["echo.default.svc.cluster.local"]
  rules:
  - backendRefs:
    - name: echo
      port: 80
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: HTTPRoute
metadata:
  name: other
  namespace: default
spec:
  parentRefs:
  - kind: Mesh
    name: istio
  hostnames: ["echo.other.svc.cluster.local"]
  rules:
  - backendRefs:
    - name: example
      port: 80
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: HTTPRoute
metadata:
  name: dual
  namespace: default
spec:
  parentRefs:
  - kind: Mesh
    name: istio
  hostnames: ["echo.default.svc.cluster.local", "echo.other.svc.cluster.local"]
  rules:
  - backendRefs:
    - name: example
      port: 80
