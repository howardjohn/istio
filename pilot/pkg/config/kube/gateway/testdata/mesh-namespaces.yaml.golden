apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: HTTPRoute/same.default
  creationTimestamp: null
  name: same-0-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - mesh
  hosts:
  - echo.default.svc.cluster.local
  http:
  - route:
    - destination:
        host: echo.default.svc.domain.suffix
        port:
          number: 80
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: HTTPRoute/other.default
  creationTimestamp: null
  name: other-0-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - mesh
  hosts:
  - echo.other.svc.cluster.local
  http:
  - route:
    - destination:
        host: example.default.svc.domain.suffix
        port:
          number: 80
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: HTTPRoute/dual.default
  creationTimestamp: null
  name: dual-0-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - mesh
  hosts:
  - echo.default.svc.cluster.local
  http:
  - route:
    - destination:
        host: example.default.svc.domain.suffix
        port:
          number: 80
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: HTTPRoute/dual.default
  creationTimestamp: null
  name: dual-1-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - mesh
  hosts:
  - echo.other.svc.cluster.local
  http:
  - route:
    - destination:
        host: example.default.svc.domain.suffix
        port:
          number: 80
---
